config/database.yml

```diff
@@ -1,18 +1,18 @@
-
+---
 development:
-  adapter: postgresql
-  database: elovation_development
-  host: localhost
-  min_messages: WARNING
+  adapter: sqlite3
```

Comment:
We don't have any stored procs etc. and postgres is an overkill for local development and testing.